x-pack/test/cases_api_integration/security_and_spaces/tests/common/internal/user_actions_get_users.ts

```diff
@@ -0,0 +1,380 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { Cookie } from 'tough-cookie';
+import { UserProfile } from '@kbn/security-plugin/common';
+import { securitySolutionOnlyAllSpacesRole } from '../../../../common/lib/authentication/roles';
+import { getPostCaseRequest } from '../../../../common/lib/mock';
+import { createCase, deleteAllCaseItems, updateCase } from '../../../../common/lib/utils';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import { getCaseUsers } from '../../../../common/lib/user_actions';
+import { loginUsers, bulkGetUserProfiles } from '../../../../common/lib/user_profiles';
+import { createUsersAndRoles, deleteUsersAndRoles } from '../../../../common/lib/authentication';
+import {
+  obsOnly,
+  obsOnlyRead,
+  obsSec,
+  obsSecRead,
+  secOnly,
+  secOnlyRead,
+  secOnlySpacesAll,
+  superUser,
+  noKibanaPrivileges,
+  globalRead,
+} from '../../../../common/lib/authentication/users';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
+  const es = getService('es');
+
+  describe('user_actions_get_users', () => {
```

Comment:
Should we delete all the case items after the tests are completed?

Reply:
You are right. I had it inside the first `describe('no profiles', ...)` but it should be in the parent.